pkg/clients/clients.go

```diff
@@ -255,5 +266,9 @@ func initTestingFramework(c *Clients) *Clients {
 		},
 	}
 
-	return AddToFrameworkScheme(apis.AddToScheme, configList, c)
+	c, err := AddToFrameworkScheme(apis.AddToScheme, configList, c)
+	if err != nil {
```

Comment:
do we need this `if`? The `AddToFrameworkScheme` returns error as `nil` if there are no errors. We can just have `return c, err`

Reply:
I missed,  I would add  `return AddToFrameworkScheme(apis.AddToScheme, configList, c)`